pkg/artifactory/resource/repository/local/local.go

```diff
@@ -50,6 +50,7 @@ type RepositoryBaseParams struct {
 	PropertySets           []string `hcl:"property_sets" json:"propertySets,omitempty"`
 	ArchiveBrowsingEnabled *bool    `hcl:"archive_browsing_enabled" json:"archiveBrowsingEnabled,omitempty"`
 	DownloadRedirect       *bool    `hcl:"download_direct" json:"downloadRedirect,omitempty"`
+	CdnRedirect            *bool    `hcl:"cdn_direct" json:"cdnRedirect,omitempty"`
```

Comment:
No need for the `hcl` tag if the CamelCase matches the snake_case. I know the rest of the fields have the `hcl` tags. They were left over from before we implement the auto conversion code.

We prefer not to use `omitempty` tag since this has been historically create unpredictable behavior.